viewreport.php

```diff
@@ -98,9 +100,14 @@
     $PAGE->set_title($reportname);
     $PAGE->set_heading( $reportname);
     $PAGE->set_cacheable( true);
-    echo $OUTPUT->header();
+    if (!$embed) {
+        echo $OUTPUT->header();
+    }
 
-    if ($hasmanageallcap || ($hasmanageowncap && $report->ownerid == $USER->id)) {
+    if (
+        !$embed
```

Comment:
This is kinda yucky formatting. Perhaps making the second line a variable and compare $embed and $var